pkg/readiness/ready_tracker_unit_test.go

```diff
@@ -62,50 +63,21 @@ var (
 		},
 	}
 
-	podGVK = schema.GroupVersionKind{Group: "", Version: "v1", Kind: "Pod"}
+	podGVK = testSyncSet.Spec.GVKs[0].ToGroupVersionKind()
```

Comment:
This should just be called "syncedGVK" if you're going to dynamically derive it, or keep "Pod" explicitly defined.

Reply:
I'll revert to podGVK since we need a pod unstructured. I had acted on this feedback before: https://github.com/open-policy-agent/gatekeeper/pull/3030#discussion_r1376928580

Reply:
+1, feel free to blame me. Tests should be as obviously correct as possible.